hpc-toolkit-blueprint/README.md

```diff
@@ -0,0 +1,35 @@
+## Description
+
+The cluster provisioning tool exposes the functionalities to provisioning a GPU
+cluster via Terraform modules. So the GPU cluster provisioning functionalities can 
+be directly integrated with HPC toolkit.
+
+This directory provides 
+
+Below is a short introduction to [HPC toolkit](https://cloud.google.com/hpc-toolkit/docs/overview) and the resource materials for it.
+
+## [HPC Toolkit](https://cloud.google.com/hpc-toolkit/docs/overview)
```

Comment:
Because it's a header, it may not be clear that this is also a hyperlink. Please add the same link to line 13 for `HPC Toolkit`